

class IssueFoundException(Exception):
    pass


class IssueCheckNotFoundException(Exception):
    pass
